### ISSUE TITLE
Questions about the batchsize

### ISSUE THREAD

**Issue by Andantino97** [MichiganCOG/TASED-Net#11]
Dear kyle,
Thanks for your excellent work.
In your code, the batchsize(minibatch maybe) was set to 8 and the pile was set to 5, to get a batchsize of 40.
However, a minibatch of 8 is too large for my GPU, and my GPU can only hold 2 minibatches. Can I change the pile to 20, to have a batchsize of 40? Could this setting get the same performance of yours? (Other parameters remain the same.)

I tried this method but it doesn't work. For the first 20 iterations, the loss goes down smoothly to 2, but then the loss began to go up and keeps a value of 3. I think there is something wrong with the training. but I don't know how to figure it out on my GPU.
Fortunately. I changed the learning rate from 0.1 to 0.01 at the beginning. and it works, but after training, I use the metrics to evaluate the model, and I didn't get as good performance as yours. Is there any way to replicate the result on my low configuration computer。。。 

Looking forward to your reply, thank you.
Best regards,
Hao.

**Comment by kylemin** [comment]
Hi,
Thank you for your interest.
The following setting resulted in a similar performance:
```
batch_size = 2
num_gpu = 1
pile = 5
it1 = 1000
it2 = 1500
num_iters = 1750
```
These parameters are not optimized at all, so you might get a better performance if you optimize them a little bit more.
Thank you.

**Comment by Andantino97** [comment]
Dear kyle,
thank you for your timely reply, and I will try it soon!

Best regards,
Hao.
